### PR TITLE
Add calendar grid

### PR DESCRIPTION
**Pull request by thecristen**
#### Summary of changes
**Asana Ticket:** [Add event listings to the grid](https://app.asana.com/0/385363666817452/1199915858612066/f)

There are a few noticeable issues which I plan to address in the 'Add toggle between views to navigation' ticket, including:

* rename URL parameter from calendar to cal
* keep all months in the left nav, not just the current one
* hide the calendar on mobile (presumably showing the list in its place)
* adding the working toggle

### CODE REVIEW

File: apps/site/lib/site_web/templates/event/index.html.eex [hunk 1]
```diff
@@ -5,19 +5,29 @@
   </header>
```
Comment (hannahpurcell) [inline apps/site/lib/site_web/templates/event/index.html.eex:5]: The calendar work does not play behave when event-page-setup script is run on this page.  (The conditional for the script is based on the `m-events-hub` selector, so using a selector from `_event_list` instead should do the trick.)

**Review by hannahpurcell — APPROVED** [review]
Looks fantastic!  One note on console errors, and one note on a couple design tweaks

File: apps/site/lib/site_web/templates/event/index.html.eex (hunk 1 above)
Comment (thecristen) [reply apps/site/lib/site_web/templates/event/index.html.eex:5]: Just updated this, and I'm having trouble reproducing so I'm hoping that means it's fixed now!

**Comment by hannahpurcell** [comment]
Oh gosh!  Sorry, I didn't notice your request to re-review.  I only saw it was approved and just kept looking past 😓  I thought the url to access the calendar was `{host}/events?calendar`, and looking at the code, that seems right to me...  did anything change that could impact my being able to access it there?

File: apps/site/lib/site_web/templates/event/index.html.eex (hunk 1 above)
Comment (hannahpurcell) [reply apps/site/lib/site_web/templates/event/index.html.eex:5]: sure is!

**Review by hannahpurcell — APPROVED** [review]
The two updates are great:  grid design looks clean, and console errors are gone

**Pull request merged**